src/test/java/seedu/address/testutil/PersonBuilder.java

```diff
@@ -49,8 +54,9 @@ public PersonBuilder(Person personToCopy) {
         phone = personToCopy.getPhone();
         email = personToCopy.getEmail();
         address = personToCopy.getAddress();
-        tags = new HashSet<>(personToCopy.getTags());
         remark = personToCopy.getRemark();
+        birthday = personToCopy.getBirthday();
```

Comment:
This is done correctly but the handling of birthday across different classes apeared to be inconsistent, sometimes Optional is involved when not needed

Reply:
Alright, most if not all of the Optionals should have been moved to the correct place by now